events/eid_al_fitr/meta.md

```diff
@@ -0,0 +1,8 @@
+---
+start_date: May 1
+end_date: May 8
+---
+
+**Eid al-Fitr**
+
+[Eid al-Fitr](https://en.wikipedia.org/wiki/Eid_al-Fitr) is celebrated after a month long fasting during the arabic month Ramadan, it begins the day after Ramadan is over and lasts three days. As soon as the Sun sets from the last day of Ramadan, Eid al-Fitr starts and the Muslims feast. Muslims also say a special prayer called the Eid prayer and food and drink are passed around in Mosques.
```

Comment:
```suggestion
[Eid al-Fitr](https://en.wikipedia.org/wiki/Eid_al-Fitr) is celebrated after a month-long dawn-to-sunset fasting of Ramadan. It falls on the first day of [Shawwal](https://en.wikipedia.org/wiki/Shawwal) in the Islamic calendar, which will change from year to year in the Gregorian calendar depending on when the new moon is sighted. The duration of this celebration also varies from culture to culture.

During Eid, Muslims will feast on delicious foods, perform special prayers, and pass around food and drink in mosques. Wish them well by greeting them with "Eid Mubarak" during this celebration!
```

I'm not that happy with this text, but here's an attached suggestion for a change.